### DISCUSSION TITLE
Is it possible to exclude some operationIds / endpoint when creating postman collection

### DISCUSSION THREAD

**Discussion by cfrapper** [apideck-libraries/portman#328] [Q&A] [answered]
Context : We have a yml file where our whole API are defined, but some of the endpoint are strictly accessible for some user (for example admin endpoint etc).

Is it possible with a portman config to decide which endpoint include in the generated postman collection ? It could automatically filter the classic "x-internal: true" ?

**Comment by cfrapper** [comment, marked as answer]
Ok sorry, I double checks on the docs and there is an option for this defined here https://github.com/apideck-libraries/portman/blob/38a07af926c50c27039fc6a59920008bca068e03/examples/cli-filtering/oas-format-filter.json#L1

  "filterFile": "./examples/cli-filtering/oas-format-filter.json",

where you can add filters

**Comment by thim81** [comment]
Hi @cfrapper 

Portman leverages [openapi-format](https://github.com/thim81/openapi-format#openapi-filter-options) for filtering capabilities, which supports filtering on: Operation ID, path, tags, ...

Have a look at the filter example for more details >> https://github.com/apideck-libraries/portman/tree/main/examples/cli-filtering
